taca/nanopore/ONT_run_classes.py

```diff
@@ -459,12 +459,33 @@ def fetch_anglerfish_samplesheet(self) -> bool:
                 raise RsyncError(
                     f"{self.run_name}: Error occured when copying anglerfish samplesheet to run dir."
                 )
+            
+    def has_fastq_output(self) -> bool:
+        """Check whether run has fastq output."""
+
+        reads_dir = os.path.join(self.run_abspath, "fastq_pass")
+
+        if os.path.exists(reads_dir):
+            return True
+        else:
+            return False
```

Comment:
Some old-school hacker hack
```suggestion
        return os.path.exists(reads_dir)
```